framework_tvb/tvb/interfaces/web/static/js/bursts_dynamic.js

```diff
@@ -534,4 +534,23 @@ function setEventsOnFormFields(param, div_id) {
 
 function prepareRefreshSubformUrl(currentElem, elementType, subformDiv) {
     return 'refresh_subform/' + dynamicPage.dynamic_gid + '/' + currentElem.value + '/' + elementType;
+}
+
+function displayDocForModel(model){
+    model_id = model.find(":selected").val();
+    $("#data_model_" + model_id.replace(/ /g,"_")).css("display", "block");
+}
+
+function fillDoc(){
+    var model = $('#model');
+    model.on("change", function(event){
+        $("#model > option").each(function(){
+            model_id = $(this).val();
+            $("#data_model_" + model_id.replace(/ /g,"_")).css("display", "none");
+        })
```

Comment:
Could this `each` and string processing be replaced by jquery selector: `div[id^="data_model_"]` ?
It should return all `div` elements with `id` property starting with `data_model_`. Is this what you want?

Reply:
Yes, I didn't know about this, but that each can be replaced with a single line: $('div[id^="data_model_"]').css("display", "none");.